package.json

```diff
@@ -1,11 +1,32 @@
 {
   "private": true,
+  "name": "material-motion-js",
   "lerna": "2.0.0-beta.30",
   "version": "1.0.0",
   "scripts": {
-    "bootstrap": "yarn; $( yarn bin )/lerna bootstrap"
+    "bootstrap": "yarn; $( yarn bin )/lerna bootstrap",
+    "test": "$( yarn bin )/karma start"
   },
   "devDependencies": {
-    "lerna": "2.0.0-beta.30"
-  }
+    "lerna": "2.0.0-beta.30",
+    "chai": "3.5.0",
+    "karma": "1.3.0",
+    "karma-chrome-launcher": "2.0.0",
+    "karma-firefox-launcher": "1.0.0",
+    "karma-mocha": "1.2.0",
+    "karma-safari-launcher": "1.0.0",
+    "karma-webpack": "1.8.0",
+    "mocha": "3.1.1",
+    "mocha-sugar-free": "1.3.1",
+    "sinon": "2.0.0-pre.3",
+    "sinon-chai": "2.8.0",
+    "ts-loader": "0.9.0",
+    "webpack": "2.1.0-beta.25"
+  },
+  "author": "The Material Motion Authors (see AUTHORS)",
+  "license": "Apache-2.0",
+  "bugs": {
```

Comment:
To save you a trip, I'm gonna clean this up myself.  Since this isn't an actual package, I want to minimize the metadata we store here.